docs/installation-docker.rst

```diff
@@ -0,0 +1,100 @@
+===================
+Docker Installation
+===================
+
+Getting your own development environment.
+
+Preparing Your System
+---------------------
+
+**Prerequisites:** You 'll need to install docker and docker-compose in your system.
+
+#. **Install prerequisites:**
+
+   Most Linux distributions come with a packaged version for docker and docker-compose.
+
+   - For debian based systems::
+
+     $ sudo apt-get install docker.io docker-compose
```

Comment:
This is just a hint for debian based systems. Since the documentation is linked a couple of lines below, I think it's OK to leave it as is.